Contents/Sketch/colors.js

```diff
@@ -7,7 +7,5 @@ var applyColors = function (newColors, docData) {
     color.alpha = newColors[i].Opacity;
     colors.push(color);
   }
-  colors = MSArray.dataArrayWithArray(colors);
-  appController.globalAssets().setPrimitiveColors(colors);
-  appController.globalAssets().objectDidChange();
```

Comment:
Nope, I was literally just hacking away to make it work. I'm making mistakes like this because I'm a noob.